### ISSUE TITLE
Show loading when the filter changes on ServiceInventory

### ISSUE THREAD

**Issue by butterybread** [inmanta/web-console#329]
## Scenario
For now no loading is shown when the filter is updated. The data just changes when the request is resolved.
This behaviour has some potential issues:
- When the filter you selected doesn't have an influence on the resulting data set, the user sees no changes.
  This can cause confusion.  -> "Is the filter applied or not? Wy is nothing happening?"
- When the user selects a new filter and the subsequent request takes a long time...
  again the user is left wandering, "What is happening?".

## Solution
When we change the filter, we should ALWAYS show the loading view (for at least 200ms? 500ms?).
This way the user is informed something is happening.

## Implementation
This should be fairly simple to add.
We need to make a differentiation between an "auto update" and a "new update".
- For "auto update", we don't want to show a loading screen because then you would see a loading screen every 5s.
- For "new update", we show a loading screen for the reasons mentioned above.

Auto updates are triggered by the SubscriptionController.
First requests for a new update are triggered on a higher level.
So these are different invocations of the same method.

So to recap, the actual change is:
When we start a new request, setState to loading. (This is just skipped now I guess)
